src/libraries/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -50,11 +50,11 @@ public partial class Socket : IDisposable
         // Our internal state doesn't automatically get updated after a non-blocking connect
         // completes.  Keep track of whether we're doing a non-blocking connect, and make sure
         // to poll for the real state until we're done connecting.
-        private bool _nonBlockingConnectInProgress;
+        private bool _pollPendingConnect;
```

Comment:
I kind of like the old field name (`_nonBlockingConnectInProgress`) better -- it conveys *why* we are polling.